packages/form-system/stories/20-System-Overview.stories.mdx

```diff
@@ -80,14 +80,14 @@ Fieldsets are groups of fields. They can be considered fields on their own as we
 partly share the normalized api via `FormControlMixin`.
 Fieldsets are at the base of:
 
-- [`LionFieldset`](?path=/docs/forms-fieldset-overview--page)
-- [`LionForm`](?path=/docs/forms-form-overview--page)
-- [`LionRadioGroup`](?path=/docs/forms-radio-group--default-story)
-- [`LionCheckboxGroup`](?path=/docs/forms-checkbox-group--default-story)
+- [LionFieldset](?path=/docs/forms-fieldset-overview--page)
+- [LionForm](?path=/docs/forms-form-overview--page)
+- [LionRadioGroup](?path=/docs/forms-radio-group--default-story)
+- [LionCheckboxGroup](?path=/docs/forms-checkbox-group--default-story)
 
 ## Other Resources
 
 - [Model Value](?path=/docs/forms-system-modelvalue--page)
 - [Formatting and parsing](?path=/docs/forms-system-formatting-and-parsing--parser)
 - [Interaction states](?path=/docs/forms-system-interaction-states--interaction-states)
-- [Validation System](?path=/docs/forms-validation-overview--page) 
+- [Validation System](?path=/docs/forms-validation-overview--page)
```

Comment:
This space in the end somehow prevented this link from not working in our extension layer...

Reply:
Hmm okay I'll look into that :|